src/test/java/com/powsybl/openloadflow/network/VoltageControlNetworkFactory.java

```diff
@@ -1383,4 +1383,29 @@ public static Network createWithShuntSharedRemoteControl2() {
 
         return network;
     }
+
+    public static Network createWithSharedRemoteControl2GensOnSameBusAnd1Extra() {
+        Network network = FourBusNetworkFactory.createWith2ReactiveControllersOnSameBusAnd1Extra();
+        Generator g1 = network.getGenerator("g1");
+        g1.setMaxP(10);
+        Generator g1Bis = network.getGenerator("g1Bis");
+        g1Bis.setMaxP(10);
+        Generator g4 = network.getGenerator("g4");
+        g4.setMaxP(20);
+        Terminal regTerminal = network.getLine("l34").getTerminal2();
+
+        g1.setRegulatingTerminal(regTerminal);
+        g1.setTargetV(1.6);
```

Comment:
functional: The 4 bus network has 1kV nominal voltage, why do you need such high unrealistic target for the purpose of reactive keys test ?